Sentry.xcodeproj/project.pbxproj

```diff
@@ -6683,6 +6676,7 @@
 		D84DAD522B17428D003CF120 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALLOW_TARGET_PLATFORM_SPECIALIZATION = YES;
```

Comment:
Please also move this to an xcconfig

Reply:
Strange, I think Xcode added this. I dont remember changing this.